.eslintrc

```diff
@@ -297,6 +298,7 @@
         "node": true
       },
       "rules": {
+        "eslint-comments/no-unused-disable": "off",
```

Comment:
Instead of disabling this rule we should just remove the offending comments because they are actually unused now.